Tasks/PipAuthenticateV1/pipauthenticatemain.ts

```diff
@@ -13,9 +14,33 @@ async function main(): Promise<void> {
 
     let internalFeedSuccessCount: number = 0;
     let externalFeedSuccessCount: number = 0;
+    let federatedFeedAuthSuccessCount: number = 0;
+
     try {
-        let internalAndExternalEndpoints: string[] = [];
+        const feedUrl = tl.getInput("feedUrl");
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
 
+        // If using feed Url and wif service connection, set the pip index url
+        if (feedUrl && entraWifServiceConnectionName)
+        {
+            tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName, feedUrl));
+            const feedTenant = await getFeedTenantId(feedUrl);
```

Comment:
feedTenant might be null or undefined here - we need to check for that

Reply:
the feed tenant can be null but is being passed as an optional parameter, if unresolved it falls back to the service connection tenant. This approach mimics our other cred providers and would unblock multi tenanted apps.